packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -109,8 +112,9 @@ export async function getPortalsTradeQuote(
     const outputToken = `${portalsNetwork}:${buyAssetAddress}`
 
     // Attempt fetching a quote with validation enabled to leverage upstream gasLimit estimate
-    const portalsTradeOrderResponse = await fetchPortalsTradeOrder({
-      sender: sendAddress,
+    const portalsTradeQuoteResponse = await fetchPortalsTradeOrder({
+      hasWallet,
+      sender: hasWallet ? sendAddress! : undefined,
```

Comment:
all of these don't need the ternary which is just setting it to basically what they would be defined as anyways.
```suggestion
      sender: sendAddress,
```

Reply:
Another one of trade vs. quote split leftovers! 269a24149f